Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM node:6.9
+FROM node:10.23.0-alpine
```

Comment:
@raffrearaUS We actually don't now if this Dockerfile is used for anything?

@NicoVogel You updated the node version to 12 in travis but here it is 10. Any reason for that?

Reply:
The Dockerfile is part of the Node template we used for the project, but it is not used for anything and can be safely removed.

Reply:
Then I would remove it as part of this PR.